### PR TITLE
cleanup/improvements

### PR DESCRIPTION
**Pull request by Juesto**
implemented refresher on modding advancements, you're welcome ;)

### CODE REVIEW

**Comment by Juesto** [comment]
reminder to squash when merging

File: StartLua.lua [hunk 1]
```diff
@@ -15,11 +15,14 @@ dofile("RunOnce.lua")
...
-   Prompt( err )
+    file2 = io.open("stderr.txt","a")
+    file2:write(os.date( "%a %b %d, %H:%M").." - "..err.."\n")
+    file2:close()
+    Prompt( err )
 end
 
 function Prompt(text)
-os.execute("@echo on&echo "..text.."&pause")
+os.execute("@echo "..text.."&timeout 6")
```
Comment (wolf109909) [inline StartLua.lua:25]: Nice error logging update, but I prefer having the error cmd windows there for me to check. also, this works as a software break if in emergency need.

**Review by wolf109909 — CHANGES_REQUESTED** [review]
This file is rather important, it's an edited version that's guaranteed to work if you just embed it into journey.exe

File: README.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 
-Embed Tick.lua.gz into Journey.exe according to this tutorial made by Alazar88:
+Modify \_\_Tick.lua.gz with our changes and embed it into Journey.exe according to this tutorial made by Alazar88:
 
 https://thatgamecompanyfan.boards.net/thread/541/pc-lua-script-editing-injection
 
-(You need to get approved first to view modding forum)
+(You need to get approved in the modding forum to view the thread)
+
+__Optional recommendations:__
```
Comment (wolf109909) [inline README.md:16]: Still I do not recommend random people to do steamless and such..

File: README.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
 # Journey_Lua_Modder
-A python GUI application aiming to make Lua injection to the game "Journey" easier.
+A python GUI application aiming to make Lua injection to the game "Journey" easier. 
 
-How to install:
+Also works in the game "Flower" with minor changes. (do not copy Tick.lua from here, instead just put the function details in the existing file)
```
Comment (wolf109909) [inline README.md:4]: I'm not deciding to make this "that game Lua modder" thing yet, but you can always just clone mine and make a flower Lua modder based on that.

File: README.md [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
-(You need to get approved first to view modding forum)
+(You need to get approved in the modding forum to view the thread)
+
+__Optional recommendations:__
+- Patch your modding exe via steamless to disable Steam launching redirection, still requires Steam running. 
+- Use a emulator when modding to avoid impacting others gameplay.
 
-run JourneyLuaModder.py
+Requires Python 3, run pythonw JourneyLuaModder.py
```
Comment (wolf109909) [inline README.md]: After having troubles with working directories and python versions, I engage all users to stick to the release version which is in exe form.

**Review by wolf109909 — APPROVED** [review]
Great logging improvements. thx.

File: StartLua.lua (hunk 1 above)
Comment (Juesto) [reply StartLua.lua:25]: this is for default, you can literally remove the timer and it'll stick.
the game suspends execution whenever the window shows up, errors will be logged anyway, we can bring back the stdout printer that creates bloat....
logs get timestampted soo

File: README.md (hunk 4 above)
Comment (Juesto) [reply README.md]: Please bundle the exe with latest python 3.8 because 3.9 drops support for windows 7

**Pull request merged**